configure.ac

```diff
@@ -110,7 +110,8 @@ AC_CHECK_HEADERS([algorithm array cassert cmath queue \
 cstdio string sys/stat.h sys/types.h ctime tuple unistd.h unordered_map \
 utility vector], [], [AC_MSG_ERROR([Missing header file])])
 
-# Check for libjudy, libgmp, libnanomsg, libpcap
+# Check for pthread, libjudy, libgmp, libnanomsg, libpcap
+AC_CHECK_LIB([pthread], [pthread_create], [], [AC_MSG_ERROR([Missing pthread library])])
```

Comment:
I was about to use the M4 `AX_PTHREAD` macro to check for pthread support. This macro checks for `libpthread` and handles the `-pthread` flag appropriately (with a special case for clang).